.github/workflows/publish_book.yml

```diff
@@ -27,11 +27,16 @@ jobs:
       with:
         path: compiled_book/
 
+    - name: Rename book to tag
+      if: startsWith(github.ref, 'refs/tags/')
+      shell: bash -l {0}
+      run: mv compiled_book/book.pdf compiled_book/${{ github.ref_name }}.pdf
```

Comment:
Given the old good bash I would use something like

`"compiled_book/introocean-en-${REF_NAME:1}.pdf"`

where `REF_NAME` is an environment variable keeping the tag name to get a release name with a prefix and remove the starting char from the tag name (assuming tag name follows the convention `v20220922`).

Not sure, however, that such syntax is allowed for this config file format.

Reply:
Should work. Want to give it a try and sent a commit to implement that?